### PR TITLE
Update problem_solving.md

### PR DESCRIPTION
**Pull request by dm-murphy**
Fixed more errors. Glad this is helpful. I will leave the PR template in the future but for this one I wanted to double check one change. For the "Understanding The Problem" section I believe the first line meant to say "Write a program" but I could be incorrect if the original "Write a problem" has a particular meaning here. Just wanted to clarify that one.

Thanks,
Daniel Murphy

### CODE REVIEW

File: ruby_programming/basic_ruby/problem_solving.md [hunk 1]
```diff
@@ -26,15 +26,15 @@ By the end of this lesson, you should be able to do the following:
 ### Understand the Problem
 The first step to solving a problem is understanding exactly what the problem is. If you don't understand the problem you won't know when you've successfully solved it and may waste a lot of time on a wrong solution.
 
-To gain clarity and understanding of the problem, write it down on paper, reword it in plain english until it makes sense to you, draw diagrams if that helps. When you can explain the problem to someone else in plain english, you understand it.
+To gain clarity and understand the problem, write it down on paper, reword it in plain english until it makes sense to you, and draw diagrams if that helps. When you can explain the problem to someone else in plain english, you understand it.
```
Comment (marvingay) [inline ruby_programming/basic_ruby/problem_solving.md:29]: The original verbiage for "understanding of the problem" is better.

File: ruby_programming/basic_ruby/problem_solving.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ What inputs will your program have? Will the user enter data or will you get inp
 The user will enter a number from the command line.
 
 What's the desired output?
-The desired output is a list of numbers from 1 to the number the user entered. But each number that is divisible by 3 will output Fizz, each number that is divisible by 5 will output "Buzz" and each number that is divisible by both 3 and 5 will output "FizzBuzz"
+The desired output is a list of numbers from 1 to the number the user entered. But each number that is divisible by 3 will output "Fizz", each number that is divisible by 5 will output "Buzz" and each number that is divisible by both 3 and 5 will output "FizzBuzz".
```
Comment (marvingay) [inline ruby_programming/basic_ruby/problem_solving.md:85]: All output terms may as well be converted to using backticks for readability.

**Comment by marvingay** [comment]
@dm-murphy Thank you for the contribution. I left a couple of comments, as when we get "revisions" or "editing" of the grammar/spelling it's often practical to make other minor adjustments that may have been missed or improvements that complement the ones provided.

Would you be up for changing the double quotes to single backticks for readability?

**Comment by dm-murphy** [comment]
@marvingay Thanks and very happy to help. I think I correctly fixed and submitted the backticks changes and reverted the one line back to the original. I'm still getting used to the Git process here.

**Comment by marvingay** [comment]
@dm-murphy Thank you so much! Perfect.

**Comment by dm-murphy** [comment]
Awesome!
